frontend/src/modules/history/components/Translation.css

```diff
@@ -3,13 +3,29 @@
     cursor: pointer;
     padding: 5px 10px 20px 10px;
     transition: all 0.1s ease-out;
+    display: flex;
 }
 
 .history .translation:hover {
     background: #4D5967;
 }
 
-.history .translation > header {
+.history .translation .avatar {
+    padding-right: 8px;
+    position: relative;
+    top: 3px;
+}
+
+.history .translation .avatar img {
+    border-radius: 4px;
+    border: 1px solid #4D5967;
```

Comment:
Let's use `#5E6475` instead (same as translation borders) and make it `#637283` on hover. Sorry for not pointing that out in the spec!